pkg/collector/corechecks/snmp/utils.go

```diff
@@ -0,0 +1,28 @@
+package snmp
+
+import "fmt"
+
+func createStringBatches(elements []string, size int) ([][]string, error) {
```

Comment:
nit: what about `stringBatchesFromStringArray` ans s/size/batchSize.
Or `splitStringArrayInBatches`

Reply:
`StringArray` is already in the signature of the function `elements []string`, I think it's redundant to have it in the function name.